dataduct/database/parsers/tests/test_create_table.py

```diff
@@ -28,3 +31,29 @@ def test_basic():
         eq_(output['exists_checks'], exists_checks)
         eq_(len(output['constraints']), 0)
         eq_(len(output['columns']), 2)
+
+    @staticmethod
+    def test_bad_input():
+        """Feeding malformed input into create table
+        """
+        query = 'CREATE TABLE orders (' +\
+                'customer_id INTEGER DISTKEY PRIMARY KEY'
+
+        try:
+            parse_create_table(query)
+            assert False
+        except ParseException:
+            pass
+
+    @staticmethod
+    def test_bad_input_in_columns():
+        """Feeding malformed input into create table
+        """
+        query = 'CREATE TABLE orders (' +\
+                'customer_id NEGATIVE DISTKEY PRIMARY KEY)'
+
+        try:
+            parse_create_table(query)
```

Comment:
Instead of this the test should be `raises(ParseException)`. Something like https://github.com/coursera/dataduct/blob/develop/dataduct/database/sql/tests/test_sql_statement.py#L32